src/js/components/Select/doc.js

```diff
@@ -221,6 +221,12 @@ export const themeDoc = {
     type: 'string | (props) => {}',
     defaultValue: undefined,
   },
+  'select.control.open': {
+    description:
+      "Any additional style for the control's open state of the Select component.",
```

Comment:
can we change it to single quote to be consistent with the rest of the doc.js file?

Reply:
Yes. I will fix.

Reply:
Apparently Prettier didn't like my escaped quote and was changing the singles to double quotes. Modified.